### PR TITLE
feat: spring batch with kubernetes scheduler

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1352

### Change description ###
feat: spring batch with kubernetes scheduler  

**Does this PR introduce a breaking change?** (check one with "x")
No

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -23,10 +23,13 @@ spring:
         jdbc:
           lob:
             non_contextual_creation: true
+  batch:
+    initialize-schema: always
   jackson.date-format: yyyy-MM-dd
   flyway:
     enabled: ${FLYWAY_ENABLED:true}
     locations: classpath:db/migration,classpath:db/postgres
+    baselineOnMigrate: true
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:33]: do we need baselineOnMigrate now?

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -3,11 +3,11 @@ server:
 
 management:
   endpoints:
-    jmx:
-      exposure:
-        include: 'health'
     web:
       base-path: "/"
+      exposure:
+        include: health, info, prometheus
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:9]: remove this

File: src/main/resources/application.yaml (hunk 1 above)
Comment (abhijitdiwan) [reply src/main/resources/application.yaml:33]: not now I think. Will see by removing

File: build.gradle [hunk 3]
```diff
@@ -169,13 +169,12 @@ jacocoTestReport {
...
+    testSourceSets  = [sourceSets.test, sourceSets.integrationTest]
     excludedClasses = [
             'uk.gov.hmcts.reform.juddata.configuration.*',
-            'uk.gov.hmcts.reform.juddata.controller.*',
             'uk.gov.hmcts.reform.juddata.JudicialApplication',
-            'uk.gov.hmcts.reform.juddata.scheduler.*',
-            'uk.gov.hmcts.reform.juddata.camel.route.*',
-            'uk.gov.hmcts.reform.juddata.camel.processor.*'
+            'uk.gov.hmcts.reform.juddata.camel.route.initializer.*',
+            'uk.gov.hmcts.reform.juddata.camel.service.EmailData.java'
```
[the remark this answers is not in the document]
Comment (abhijitdiwan) [reply build.gradle:177]: initialisers we need to do as they are just starting routes  & email data has been refactored in other PR.

[the remark this answers is not in the document]
Comment (shashank-rastogi) [reply build.gradle]: add comment in the gradle file

File: src/main/resources/application.yaml (hunk 2 above)
Comment (abhijitdiwan) [reply src/main/resources/application.yaml]: done

File: src/main/resources/application.yaml (hunk 1 above)
Comment (abhijitdiwan) [reply src/main/resources/application.yaml]: done

File: build.gradle (hunk 3 above)
[the remark this answers is not in the document]
Comment (abhijitdiwan) [reply build.gradle]: covered intailizer

**Pull request merged**